docs/kb/api-version-3/export.md

```diff
@@ -39,11 +39,15 @@ A procedure is a query and execution plan used to retrieve the data. Each object
 
 ## Visitor Activity
 
-## filter_by_created_at
+### Fields
+
+Select the visitor activity fields that need to be exported. [Following](../object-field-references#visitor-activity) are the fields that are available for visitor activity. The value for `fields` must be an array of strings of the available fields. 
```

Comment:
The reference link is redirected to a kb article, and that loses the anchor argument, leaving the page all the way up, in Account